pmoves/services/replay-compressor/app.py

```
@@ -0,0 +1,282 @@
+        log.info("restored s3://%s/%s (%d bytes)", S3_BUCKET, key, len(data))
+        return data
+    except Exception as exc:
+        log.warning("restore_payload failed: %s", exc)
+        return None
+
+
+async def publish(nc: NATSClient, subject: str, data: Dict[str, Any]) -> None:
+    try:
+        await nc.publish(subject, json.dumps(data).encode())
+        log.info("published %s", subject)
+    except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)